src/content/events/2023-05-09_CEPLAS-StartYourARC-Series.md

```diff
@@ -0,0 +1,48 @@
+---
+title: CEPLAS Start Your ARC Workshop Series
+category: Training
+excerpt: 'Learn how to move your datasets into ARCs, share them via the DataHUB, and annotate them with metadata.'
+when:
+  - 
+    start: 2023-05-09T10:00:00
+    end: 2023-05-09T11:30:00
+  - 
+    start: 2023-05-31T10:00:00
+    end: 2023-05-31T11:30:00
+  - 
+    start: 2023-06-14T10:00:00
+    end: 2023-06-14T11:30:00
+mode: On-site
+audience: [Users]
+location:
+  short: HHU, ZSL 22.07, Room U1.025
+  address: HHU, ZSL 22.07, Room U1.025
```

Comment:
Not used in frontmatter?

Reply:
address you mean? 
I know, but we need it somewhere

Reply:
Agreed.

Reply:
Btw. short and address exist in parallel in some other events, see hackathons